### PR TITLE
Make scripts to run the generator phases instead of running them automatically

### PR DESCRIPTION
**Pull request by matozoid**
By popular request :)

### CODE REVIEW

File: run_core_generators.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```
Comment (ftomassetti) [inline run_core_generators.sh:1]: What about windows?

File: run_core_generators.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+# Runs all the code generators.
+# If the node structure was changed, run_metamodel_generator.sh first!
+
+pushd javaparser-core-generators
```
Comment (ftomassetti) [inline run_core_generators.sh]: Maybe pushd and popd are not so commonly used and they could deserve a comment

**Review by ftomassetti — COMMENTED** [review]
Also, should we add some comment in the documentation?

File: pom.xml [hunk 3]
```diff
@@ -3,8 +3,6 @@
     <modules>
         <module>javaparser-core</module>
         <module>javaparser-testing</module>
-        <module>javaparser-core-generators</module>
```
Comment (ftomassetti) [inline pom.xml:6]: Ok, we do not want to distribute the generators but will IDE consider this modules if we remove them from the POM?

File: run_core_generators.sh (hunk 1 above)
Comment (matozoid) [reply run_core_generators.sh:1]: It's best if a Windows user contributes those, I think.

File: pom.xml (hunk 3 above)
Comment (matozoid) [reply pom.xml:6]: Well, they are not really modules anymore. They are now stand-alone projects that use the `javaparser-parent` module as their parent. So I guess in intellij for example, you would have to treat them as separate projects.

File: run_core_generators.sh (hunk 2 above)
Comment (matozoid) [reply run_core_generators.sh]: I documented the whole thing now.

**Pull request merged**